src/main/java/com/redhat/devtools/lsp4ij/LanguageServiceAccessor.java

```diff
@@ -584,4 +599,56 @@ private void disposeAllServers() {
         }
     }
 
+    /**
+     * Returns the project-level language server accessor's modification tracker.
+     *
+     * @return the project-level language server accessor's modification tracker
+     */
+    @NotNull
+    @ApiStatus.Internal
+    public SimpleModificationTracker getModificationTracker() {

```

Comment:
Needed in `EditorBehaviorFeature#isEnabled()` for a fallback caching dependency.